src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/ColorPicker.xaml

```diff
@@ -508,7 +510,9 @@
                                 <Setter Property="Template">
                                     <Setter.Value>
                                         <ControlTemplate TargetType="Thumb">
-                                            <Border BorderBrush="{ThemeResource SliderThumbBorderBrush}"
+                                            <Border 
+                                                win:BorderBrush="{ThemeResource SliderThumbBorderBrush}"
+                                                not_win:BorderBrush="{ThemeResource ControlStrokeColorDefaultBrush}"
```

Comment:
Do we understand what will need to be fixed in future to remove this Uno-specific workaround?

Reply:
To make the workaround less pronounced, we will be able to use the `FauxLinearGradientBorder` PR (gotta get back to it now that we refactored border layer handling). However, the proper fix would be if WASM and iOS supported transforms on linear gradients on borders, which is unfortunately not the case. On iOS it might be possible, but didn't work out last time I was trying it.